scripts/dts/gen_defines.py

```diff
@@ -444,6 +444,20 @@ def irq_name_alias(irq, cell_name):
             alias += "_" + str2ident(cell_name)
         return alias
 
+    def map_arm_gic_irq_type(irq, irq_num):
+        # Maps ARM GIC IRQ (type)+(index) combo to linear IRQ number
+        if "type" not in irq.data:
+            err("Expected binding for {!r} to have 'type' in *-cells"
+                    .format(irq.controller))
```

Comment:
Could align it like this just for consistency:

```python
err("Expected binding for {!r} to have 'type' in *-cells"
    .format(irq.controller))
```

Reply:
Could maybe say `interrupt-cells` instead of `*-cells`, since it's known here.

Reply:
Fixed.